### PR TITLE
SIG Release 2022 Annual Report

### LINKED ISSUE #7112: 2022 Annual Report: SIG Release
Chairs: @jeremyrickard @justaugustus @saschagrunert 
Liaison: @palnabarun

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-release/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-release/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig release
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by jeremyrickard**
This PR adds the SIG Release 2022 Annual Report. 

**Which issue(s) this PR fixes**:

Fixes #7112

### CODE REVIEW

File: sig-release/annual-report-2022.md
Comment (saschagrunert) [inline sig-release/annual-report-2022.md:7]: Do we want to add https://github.com/kubernetes/enhancements/issues/3027 and https://github.com/kubernetes/enhancements/issues/2853 as well?

**Comment by jeremyrickard** [comment]
@saschagrunert  @palnabarun  please take a look.

File: sig-release/annual-report-2022.md
Comment (palnabarun) [inline sig-release/annual-report-2022.md:34]: Are there issues that can be linked which require attention? This will raise public awareness to the areas of each of these projects needing help.

File: sig-release/annual-report-2022.md
Comment (palnabarun) [inline sig-release/annual-report-2022.md:41]:
```suggestion
   In support of better understanding the diversity of the release team, [kubernetes-sigs/release-team-shadow-stats](https://github.com/kubernetes-sigs/release-team-shadow-stats) was begun to provide better reporting and visibility on release team metrics. For our [KubeCon EU SIG update](https://www.youtube.com/watch?v=qhQYu077zZU), we also presented a historical breakdown of the location of release team members and the geographic distribution of the release teams. 
```

File: sig-release/annual-report-2022.md
Comment (palnabarun) [inline sig-release/annual-report-2022.md:88]: Can we point to some docs/issues for these asks from end users/companies/vendors?

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md:7]: That's a good suggestion, I will add it!

File: sig-release/annual-report-2022.md
Comment (puerco) [inline sig-release/annual-report-2022.md:12]: I'm wondering if we should highlight here or in # 1 the work to keep the image promotion process working under the registry changes. Last year the promoter changes represented a big chunk of the engineering time we put into SIG Release during the second half of the year. We had a lot of releases that demanded a lot of time from the release managers, often spending several days promoting the images.

File: sig-release/annual-report-2022.md
Comment (puerco) [inline sig-release/annual-report-2022.md:128]:
```suggestion
- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
```

File: sig-release/annual-report-2022.md
Comment (saschagrunert) [reply sig-release/annual-report-2022.md:34]: I'm not sure we should link the plain repositories here, we have a bunch of larger efforts tracked in keps as well as some good first ones. We probably need most help in graduating the enhancements and everything belonging to that.

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md:13]: added text about this.

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md]: I cleaned this up, we don't have specific docs/asks and I think this is specifically indicating full-time work, which I don't think we have direct need of currently.

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md]: Updated!

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md]: Updated.

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md:7]: Updated!

**Comment by jeremyrickard** [comment]
@saschagrunert @palnabarun @puerco  please take a look, I have incorporated your feedback.

**Review by xmudrii — COMMENTED** [review]
Some nits, but other than that, very well done!

File: sig-release/annual-report-2022.md
Comment (jeremyrickard) [reply sig-release/annual-report-2022.md:7]: oh that is a very good one to include, thank you.

**Comment by jeremyrickard** [comment]
@xmudrii thank you for the review! I've incorporated your feedback, thanks for all the great suggestions!

**Review by xmudrii — APPROVED** [review]
Thank you for taking care of this report @jeremyrickard!
/lgtm
/hold
for other folks to take a look

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by palnabarun** [comment]
Thanks everyone for the annual report! 🎉 

/lgtm
/approve

/hold cancel

**Pull request merged**
